test/mix/tasks/phoenix.new_test.exs

```diff
@@ -109,6 +109,7 @@ defmodule Mix.Tasks.Phoenix.NewTest do
       assert_file "photo_blog/priv/static/images/phoenix.png"
       assert_file "photo_blog/web/static/vendor/phoenix.js"
       assert_file "photo_blog/web/static/js/app.js"
+      assert_file "photo_blog/web/static/css/_bootstrap.scss"
```

Comment:
It's there to make sass ignore the file. So it should unless we want it to be built.